src/stylesheets/components/_title.scss

```diff
@@ -1,3 +1,18 @@
 .title {
-  
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
```

Comment:
Hmm, I don't understand what's the purpose of the stuff in this situation. I don't need it. Replace with `margin: 0 auto;`.